src/Uno.UWP/UI/Core/CoreWindow.Android.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using Windows.Devices.Input;
+
+namespace Windows.UI.Core;
+
+internal sealed partial class CoreWindowExtension : ICoreWindowExtension
+{
+	public CoreCursor PointerCursor { get => throw new NotImplementedException(); set => throw new NotImplementedException(); }
+
+	public void ReleasePointerCapture(PointerIdentifier pointer) => throw new NotImplementedException();
+	public void SetPointerCapture(PointerIdentifier pointer) => throw new NotImplementedException();
```

Comment:
I think that Android allows capture, so it's probably just a "TODO", but in all cases you should not throw an exception here. It's used by a lot of controls (like `ButtonBase`), throwing here will break internal state.